src/components/activityEventElems/ActivityElement/ActivityElement.tsx

```diff
@@ -87,27 +124,177 @@ function Subject({ subject }: { subject: string | JSX.Element | null }) {
   )
 }
 
-export function ActivityEvent({
+function _ActivityEvent({
   header,
   subject,
-  event,
+  details,
   extra,
+  style,
 }: {
-  header: string
+  header: string | JSX.Element | null
   subject: string | JSX.Element | null
   extra?: string | JSX.Element | null
-  event: ActivityElementEvent
+  details: Partial<ActivityElementEvent>
+  style?: CSSProperties
 }) {
   return (
-    <>
+    <div style={style}>
       <DetailsContainer>
         <div>
           <Header header={header} />
           <Subject subject={subject} />
         </div>
-        <SideDetails event={event} />
+        <SideDetails details={details} />
       </DetailsContainer>
       {extra ? <ExtraContainer>{extra}</ExtraContainer> : null}
-    </>
+    </div>
+  )
+}
+
+export function ActivityEvent({
+  event,
+  style,
+}: {
+  event: ProjectEvent
+  style?: CSSProperties
+}) {
+  // TODO: we should use v2 where we can, or a better context
+  const { tokenSymbol } = useContext(V1ProjectContext)
```

Comment:
hmmm, we could actually store the symbol in the graph event

another context might also be a good solution, though i'd love to avoid scope creeping this PR any more lol